$MirOS: ports/print/teTeX/base/patches/patch-libs_libwww_aclocal_m4,v 1.1.7.1 2005/03/18 15:51:08 tg Exp $
$OpenBSD: patch-libs_libwww_configure,v 1.1 2003/02/18 16:04:50 sturm Exp $
# this has to fail, because bsd has timezone() returning char*
# while libwww really wants a long int timezone variable
--- libs/libwww/aclocal.m4.orig	Sun Mar  4 23:46:21 2007
+++ libs/libwww/aclocal.m4	Sun Mar  4 23:46:21 2007
@@ -464,7 +464,13 @@ fi
 dnl Borrowed from the Python distribution...
 AC_MSG_CHECKING(for time.h that defines altzone)
 AC_CACHE_VAL(ac_cv_extern_altzone,
-[AC_TRY_COMPILE([#include <time.h>], [return altzone;],
+[AC_TRY_COMPILE([#include <time.h>], [
+#ifdef __OpenBSD__
+#error libwww wants long int, we have char *
+#else
+return altzone;
+#endif
+],
   ac_cv_extern_altzone=yes,
   ac_cv_extern_altzone=no)])
 AC_MSG_RESULT($ac_cv_extern_altzone)
